perimeter/shared_access.sp

```diff
@@ -22,7 +34,11 @@ benchmark "shared_access" {
   documentation = file("./perimeter/docs/shared_access.md")
   children = [
     benchmark.ram_shared_access,
-    benchmark.shared_access_settings
+    benchmark.shared_access_settings,
+    benchmark.resource_policy_shared_accounts_access,
```

Comment:
Instead of having these top level, can you please create a new benchmark, `benchmark.resource_policy_shared_access`, and then put these 4 new benchmarks under that one? This is a similar structure to the one in `public_access.sp`, and allows a user to easily run all resource policy benchmarks, e.g., `steampipe check benchmark.resource_policy_shared_access`.

Reply:
Done